packages/rancher-logging/generated-changes/patch/templates/_helpers.tpl.patch

```diff
@@ -34,3 +34,58 @@
 +{{- define "windowsInputTailMount" -}}
 +{{- (include "windowsPathPrefix" .) | replace "C:" "" -}}
 +{{- end -}}
++
```

Comment:
This file should reflect this discussion: https://github.com/rancher/charts/pull/1223/commits/dd1b289ef2618c1c158274e9f675673c402294cb#r638422713